source/DataFields.mc

```diff
@@ -368,6 +370,14 @@ class DataFields extends Ui.Drawable {
 		var unit;
 
 		switch (type) {
+			// SG Addition
+			case FIELD_TYPE_PULSE_OX:
+				activityInfo = Activity.getActivityInfo();
+				sample = activityInfo != null and activityInfo has :currentOxygenSaturation ? activityInfo.currentOxygenSaturation : null;
+				if (sample != null) {
+					value = sample.format(INTEGER_FORMAT);
```

Comment:
Do you think the PulseOx value should be shown with a trailing "%", like Humidity and Battery?

Reply:
Would make sense, yes since it's a percentage

Reply:
Done